doc/conf.py

```diff
@@ -0,0 +1,325 @@
+# -*- coding: utf-8 -*-
+#
+# autoreject documentation build configuration file, created by
+# sphinx-quickstart on Mon May 23 16:22:52 2016.
```

Comment:
watchout you've got a few of these. Make sure you don't refer to other projects here.

Reply:
oups, thanks for the hint (do you see any other false references, I think I removed everything else). Also, the file I used as reference template was from the mne-hcp repo. So perhaps we should have a look at that repo as well.